scripts/run_integration_tests.sh

```diff
@@ -38,9 +38,10 @@ PARAMETERS_SETS=("python27" "python36" "python37" "python38")
 if [ -z "$RUNTIME_PARAM" ]; then
     echo "Python version not specified, running for all python versions."
 else
-    echo "Python version is specified: $RUNTIME_PARAM"
-    PARAMETERS_SETS=(python${RUNTIME_PARAM})
-    BUILD_LAYER_VERSION=python$RUNTIME_PARAM[1]
+    RUNTIME_PARAM_NO_DOT=$(echo $RUNTIME_PARAM | sed 's/\.//')
+    echo "Python version is specified: $RUNTIME_PARAM_NO_DOT"
```

Comment:
Nit: I think we would probably want to log the version number with the dot here, right? `Python version is specified: 3.7` rather than `Python version is specified: 37`

Reply:
✅